common/stream-model.cpp

```diff
@@ -779,177 +773,242 @@ namespace rs2
+                                        "When Post-Processing is enabled, the actual display size of the frame may "
+                                        "differ from original capture size" } );
+        }
+        stream_details.push_back(
+            { "Pixel Format", rsutils::string::from() << rs2_format_to_string( profile.format() ), "" } );
+
+        stream_details.push_back(
+            { "Hardware FPS",
+              rsutils::string::from() << std::setprecision( 2 ) << std::fixed << fps.get_fps(),
+              "Hardware FPS captures the number of frames per second produced by the device.\n"
+              "It is possible and likely that not all of these frames will make it to the application." } );
+
+        stream_details.push_back(
+            { "Viewer FPS",
+              rsutils::string::from() << std::setprecision( 2 ) << std::fixed << view_fps.get_fps(),
+              "Viewer FPS captures how many frames the application manages to render.\n"
+              "Frame drops can occur for variety of reasons." } );
+
+        stream_details.push_back( { "", "", "" } );
+    }
 
-                    stream_details.push_back({ "Display Size",
-                        rsutils::string::from() << size.x << " x " << size.y,
-                        "When Post-Processing is enabled, the actual display size of the frame may differ from original capture size" });
-                }
-                stream_details.push_back({ "Pixel Format",
-                    rsutils::string::from() << rs2_format_to_string(profile.format()), "" });
+    void stream_model::draw_stream_metadata( const double timestamp,
+                                            const rs2_timestamp_domain timestamp_domain,
+                                            const unsigned long long frame_number,
+                                            stream_profile profile,
+                                            rs2::float2 original_size,
+                                            const rect &stream_rect )
+    {
+        std::vector< attribute > stream_details;
 
-                stream_details.push_back({ "Hardware FPS",
-                    rsutils::string::from() << std::setprecision(2) << std::fixed << fps.get_fps(),
-                    "Hardware FPS captures the number of frames per second produced by the device.\n"
-                    "It is possible and likely that not all of these frames will make it to the application." });
+        create_stream_details( stream_details, timestamp, timestamp_domain, frame_number, profile, original_size );
 
-                stream_details.push_back({ "Viewer FPS",
-                    rsutils::string::from() << std::setprecision(2) << std::fixed << view_fps.get_fps(),
-                    "Viewer FPS captures how many frames the application manages to render.\n"
-                    "Frame drops can occur for variety of reasons." });
+        const std::string no_md = "no md";
```

Comment:
Please verify that the use case of not having metadata looks OK too.
You can use @remibettan help to remove the metadata from the registry and verify it looks OK as it was on an old viewer code